data/noiseprofiles.json

```diff
@@ -3820,6 +3820,51 @@
       ]
     },
     {
+      "maker": "Pentax",
+      "models": [
+        {
```

Comment:
And, the noise profile is already a pr https://github.com/darktable-org/darktable/pull/1275/files